src/graphql-config/graphql-config.service.ts

```diff
@@ -0,0 +1,41 @@
+import type { ApolloDriverConfig } from '@nestjs/apollo';
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import type { GqlOptionsFactory } from '@nestjs/graphql';
+import { ApolloServerPluginLandingPageLocalDefault } from 'apollo-server-core';
+
+import type { EnvironmentVariables } from '../configuration/EnvironmentVariables';
+import { IBAN, Money } from '../shared/scalars';
+
+const mocks = {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  DateTime: () => {
+    return new Date();
+  },
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  String: () => {
+    return 'string-mock';
+  },
+};
```

Comment:
It's better to move mocks to another file.

Reply:
I didn't understand what is the usage of this piece of code. it should be permanent?

Reply:
Me neither, @MohammadForoutan created this one, it must be for a reason. Let's keep it for now in a separate file.